### PR TITLE
refs #60: Redmine-Git-Commit

### PR DESCRIPTION
**Pull request by mgdesign**
refs #60: Redmine-Git-Commit

### CODE REVIEW

File: src/Sparkfabrik/Command/Redmine/RedmineGitCommitCommand.php [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+                $story_name = iconv('UTF-8', 'ASCII//IGNORE', $story_name);
+            }
+        }
+        $story_name = preg_replace('/[a-z0-9 ]{0,}[- ]{0,}/', '', $story_name, 1);
+
+        // Output to the user the current status of informations.
+        $output->writeln('<info>Issue Number</info>: ' . $issue);
+        $output->writeln('<info>Jira Story Code</info>: ' . $story);
+        $output->writeln('<info>Story Name</info>: ' . $story_name);
+
```
Comment (paolomainardi) [inline src/Sparkfabrik/Command/Redmine/RedmineGitCommitCommand.php:90]: Seems that till here the code is a copy/paste from the gitBranchCommand, would be right to extend from the same base class and extract methods.

**Comment by paolomainardi** [comment]
@mgdesign i want to merge this.

Can you describe me how it works ?

**Comment by mgdesign** [comment]
@paolomainardi 
Basically che command retrieves the details of the issue from Redmine, asks the user for a confirmation before building a commit message base, then will ask the user to complete it.
If something is wrong with the issue details, the command should abort.

In particular:
`./spark.php redmine:git:commit ####` will only print the commit message, then it's up to the user to do the commit command
`./spark.php redmine:git:commit #### -e` will do exactly the same but will execute the commit too
`./spark.php redmine:git:commit #### -e --no-verify` will skip the phpcs check

**Comment by paolomainardi** [comment]
:+1:

**Comment by paolomainardi** [comment]
thanks @mgdesign merged!

**Pull request merged**